web/viewer.css

```diff
@@ -584,6 +584,15 @@ body {
   inset-inline-end: calc(var(--editor-toolbar-base-offset) + 56px);
 }
 
+#editorHighlightParamsToolbar {
+  inset-inline-end: calc(var(--editor-toolbar-base-offset) + 84px);
+}
+
+#editorFreeTextParamsToolbar {
+  inset-inline-end: 124px;
+  background-color: var(--toolbar-bg-color);
+}
+
```

Comment:
This entire block should be removed, since it breaks positioning of the FreeText-toolbar; possibly a rebase error?